.idea/codeStyles/codeStyleConfig.xml

```diff
@@ -1,5 +1,6 @@
 <component name="ProjectCodeStyleConfiguration">
   <state>
     <option name="USE_PER_PROJECT_SETTINGS" value="true" />
+    <option name="PREFERRED_PROJECT_CODE_STYLE" value="Project copy" />
   </state>
-</component>
+</component>
```

Comment:
Remove this file from your commit, you can select/deselect which files you want to commit in android studio.

Reply:
okay

Reply:
You have marked it as resolved but it has not been resolved.

Also, if you are not able to remove this file, then maybe correct this file by adding EOF

Reply:
okay